### ISSUE TITLE
Support cider-nrepl 0.21.0

### ISSUE THREAD

**Issue by liquidz** [liquidz/vim-iced#91]
https://github.com/clojure-emacs/cider-nrepl/blob/master/CHANGELOG.md#0210-2019-02-11

**Comment by liquidz** [comment]
Failed to call iced-nrepl op in CLJS session.

Leiningen: 2.9.0
cider-nrepl: 0.21.0

```
Exception in thread "nRepl-session-9587c2c2-746c-4fc7-b8a4-038e2a314996" java.lang.IllegalArgumentException: No implementation of method: :send of protocol: #'nrepl.transport/Transport found for class: nil
        at clojure.core$_cache_protocol_fn.invokeStatic(core_deftype.clj:583)
        at clojure.core$_cache_protocol_fn.invoke(core_deftype.clj:575)
        at nrepl.transport$eval648$fn__649$G__639__656.invoke(transport.clj:16)
        at nrepl.middleware.print$replying_PrintWriter$fn__1105.invoke(print.clj:115)
        at nrepl.middleware.print.proxy$java.io.Writer$ff19274a.write(Unknown Source)
        at java.io.BufferedWriter.flushBuffer(BufferedWriter.java:129)
        at java.io.BufferedWriter.flush(BufferedWriter.java:253)
        at java.io.PrintWriter.flush(PrintWriter.java:320)
        at clojure.core$flush.invokeStatic(core.clj:3711)
        at clojure.core$prn.invokeStatic(core.clj:3721)
        at clojure.core$prn.doInvoke(core.clj:3714)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.core$apply.invokeStatic(core.clj:665)
        at clojure.core$println.invokeStatic(core.clj:3733)
        at clojure.core$println.doInvoke(core.clj:3733)
        at clojure.lang.RestFn.invoke(RestFn.java:408)
        at cljs.repl$display_error.invokeStatic(repl.cljc:477)
        at cljs.repl$display_error.invoke(repl.cljc:470)
        at cljs.repl$repl_caught.invokeStatic(repl.cljc:821)
        at cljs.repl$repl_caught.invoke(repl.cljc:812)
        at cider.piggieback$repl_caught.invokeStatic(piggieback.clj:105)
        at cider.piggieback$repl_caught.invoke(piggieback.clj:98)
        at cider.piggieback$do_eval.invokeStatic(piggieback.clj:265)
        at cider.piggieback$do_eval.invoke(piggieback.clj:227)
        at cider.piggieback$evaluate.invokeStatic(piggieback.clj:273)
        at cider.piggieback$evaluate.invoke(piggieback.clj:271)
        at clojure.lang.Var.invoke(Var.java:384)
        at cider.piggieback$wrap_cljs_repl$fn__1844$fn__1847$fn__1848.invoke(piggieback.clj:305)
        at cider.piggieback$enqueue$fn__1816.invoke(piggieback.clj:195)
        at clojure.lang.AFn.run(AFn.java:22)
        at nrepl.middleware.session$session_exec$main_loop__1340$fn__1344.invoke(session.clj:171)
        at nrepl.middleware.session$session_exec$main_loop__1340.invoke(session.clj:170)
        at clojure.lang.AFn.run(AFn.java:22)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by liquidz** [comment]
`iced-lint-file` and `iced-refactor-thread-first` op works fine.
`iced-format-code-with-indens` and `iced-calculate-indent-level` op does not work...

**Comment by liquidz** [comment]
I found the cause. iced-nrepl did not matter.

If an evaluation error occurs in the [**cloned**](https://nrepl.org/nrepl/0.6.0/ops.html#_clone) cljs session, this exception occurs.
I have no idea the root cause, but no exception occurs with **original** cljs session.

So I adjust the order of session cloning when switching to cljs session as a workaround.

**Comment by liquidz** [comment]
Fixed by #93
